pgpy/pgp.py

```diff
@@ -1259,6 +1259,15 @@ def is_expired(self):
         return False
 
     @property
+    def is_revoked(self):
+        """``True`` if this key has a KeyRevocation signtuare"""
+        for sign in self._signatures:
+            if sign.type == SignatureType.KeyRevocation:
```

Comment:
here subkeys should be verified and the expiration checked. that is done in #207 but the actual `is_revoked` propery is missing. furthermore, signature should actually be verified, something that is not done in #207 either.